godockerize.go

```diff
@@ -137,6 +148,13 @@ func doBuild(c *cli.Context) error {
 	if len(expose) != 0 {
 		fmt.Fprintf(&dockerfile, "  EXPOSE %s\n", strings.Join(sortedStringSet(expose), " "))
 	}
+	if user != "" {
+		fmt.Fprintf(&dockerfile, "  RUN addgroup -S %s && adduser -S -G %s -h /home/%s %s\n", user, user, user, user)
+		for _, userDir := range userDirs {
+			fmt.Fprintf(&dockerfile, "  RUN mkdir -p %s && chown -R %s:%s %s\n", userDir, user, user, userDir)
+		}
```

Comment:
minor, but could you rather create one large `RUN` command, rather than several. Just optimizing how the docker layers are generated. I am thinking a simple building up of commands, and then just `strings.Join(cmds, " && ")`